backend/api/Database/Models/Robot.cs

```diff
@@ -32,6 +32,8 @@ public class Robot
 
         public float BatteryLevel { get; set; }
 
+        public bool? ObstacleDectected { get; set; }
```

Comment:
Is this a bit specific to add it as a column to the robot table? Is there a more general status object we want to use instead?